### DISCUSSION TITLE
SDDM - not accepting password with ML4W script

### DISCUSSION THREAD

**Discussion by aecardenasfl** [mylinuxforwork/dotfiles#138] [General]
I had previously set another greeter for my Arch installation (greetd) and when using the ML4W script when it prompted to remove the older display manager I choose to remove it and then install SDDM with the script. Everything looks fine but after rebooting it does not accept my password. I have to login via TTY and then it works fine. I manually removed it and installed it with pacman and now is working fine (without any theme). Would appreciate if anyone can point me in the right direction to have it working. thanks!

**Comment by mylinuxforwork** [comment]
Can you check which keyboard layout sddm is using? If you're using a another layout then you aleays type the wrong password.

**Reply by aecardenasfl** [reply]
that was my initial thought but it is using US. I tested with no special characters and is the same, but through tty it is accepting them. 
There is also a related issue with wlogout that when I clicked the "exit" option it didn't recognize the display manager so it crashes;
I see that it runs an "exit" script and it runs killall -9 Hyprland so I replaced it with loginctl terminate-user $USER to test, it logs me out but it took me back to TTY login instead of sddm. 

I tried uninstalling it, disabling the sddm.service, reinstalling, enabling the sddm.service and now it works, but it doesn't apply the theme back, just shows the default one. 

I think it might be a previous config that I had somewhere or a permissions settings that I'm missing. I was using before greetd and for that one it uses a user that needs to be added to the "video" group, my guess is something similar is missing in my config.

**Reply by mylinuxforwork** [reply]
Yes. This could be the case. Thanks again for providing your observations.

**Comment by aecardenasfl** [comment]
I finally found out why this was not working... it was the sugar-candy theme! 

in the theme.conf there's this line:

AllowBadUsernames="false"

my username has an uppercase at the begining, and this does not allow uppercase in usernames. This shows the user capitalized, but takes it all lowercase. I set it to "true" and now shows the user as-is in the system, and also allows uppercases in usernames, thus accepting my user and password. so I should either make my username all lowercase, or set this to true xD
